src/components/Editor/CallSites.js

```diff
@@ -161,6 +166,7 @@ class CallSites extends Component {
             key: index,
             callSite,
             editor,
+            sourceId,
```

Comment:
lets pass the selectedSource down if possible. I think that would be a little more useful